.github/actions/generate-pot-file/action.yml

```diff
@@ -17,7 +17,7 @@ runs:
             run: |
                 npm install -g npm@7
                 npm ci
-                npm run dev
+                npm run build # Must be built for production before generating POT file.
```

Comment:
@kjohnson Honestly, I don't think we need the comments. Building for production doesn't seem strange at all. It was weirder before that we were building for dev. I think we can just switch to build and let the step name explain it from there.

Reply:
Resolved by https://github.com/impress-org/givewp-github-actions/pull/4/commits/d66836abe926f4f9a159b98c3e7a355f52b48c6c.